docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/bitbucket/index.md

```diff
@@ -45,6 +45,8 @@ kind: Secret
 metadata:
   name: bb-wh-secret
   namespace: kargo-demo
+  labels:
+    kargo.akuity.io/cred-type: generic
```

Comment:
Thank you for this! I was going to open a PR to update all of these and you saved me the trouble!